django_celery_token_bucket/bucket.py

```diff
@@ -19,9 +19,10 @@ class TokenBucket:
     """
 
     name: str
-    schedule: schedules.crontab
     amount: int
     maximum: int
+    schedule: schedules.crontab = None
+    interval_in_seconds: int = None
```

Comment:
could you add information on `interval_in_seconds`  to the string doc and the Readme?